transformer_engine/pytorch/ops/basic/dropout.py

```diff
@@ -25,9 +26,10 @@ class Dropout(BasicOperation):
 
     """
 
-    def __init__(self, p: float) -> None:
+    def __init__(self, p: float, otype: Optional[torch.dtype] = None) -> None:
```

Comment:
This name is non-standard. Most PyTorch ops would name this `dtype`:
```suggestion
    def __init__(self, p: float, dtype: Optional[torch.dtype] = None) -> None:
```

Reply:
TE ops don't actually support arbitrary usage of `Float8TensorBase`. Ops can request internal tensors (by setting `quantizer.internal=True`), but they shouldn't push internal tensors onto ops that don't support them. This is important so that ops can have naive fallback implementations using `__torch_dispatch__`.